pkg/apis/apirator/v1alpha1/apimock_types.go

```diff
@@ -105,3 +127,35 @@ type APIMockList struct {
 func init() {
 	SchemeBuilder.Register(&APIMock{}, &APIMockList{})
 }
+
+// ================================================================
+//                        Domain functions
+// ================================================================
```

Comment:
This is nice! :smiley:

Reply:
I think `APIMock#ExposeInIngress`,  `APIMock#AnnotatePorts`, `APIMock#AnnotateClusterIP` are domain functions too.